### PR TITLE
feat: SecuritySystem node

### PR DESCRIPTION
**Pull request by Hasan-Baig**
All functionality works in terms of HW to SW.

### CODE REVIEW

File: securitysystem/pantilt.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+#                         type=int,
+#                         required=True,
+#                         metavar='<owner_room>',
+#                         help='Specify tilt angle between 0° to 180°')
+
+#     args = parser.parse_args()
+#     return args
+
+
+if __name__ == '__main__':
```
Comment (Hasan-Baig) [inline securitysystem/pantilt.py:151]: might change (NOT READY TO MERGE):
> could change to slider input
> may delete this class and directly add to the front end webpage

File: securitysystem/pantilt.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    except KeyboardInterrupt:
+        logging.info('Exiting')
+    except BaseException as e:
+        logging.error('An error or exception occurred: ' + str(e))
+    finally:
+        servo_test.close_servo()
+        GPIO.cleanup()
+        
+
+# def parse_args():
```
Comment (marianarafaelwhite) [inline securitysystem/pantilt.py]: Are these lines going to be removed or updated later?

File: securitysystem/thingspeakreader.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+"""
+Reading data collected from ThingSpeak
+"""
+import requests
+import json
+import logging
+import argparse
+import constants as c
+
+class ThingSpeakReader():
```
Comment (marianarafaelwhite) [inline securitysystem/thingspeakreader.py:11]: I wonder how different our ThingSpeak classes are... If we have time to compare we might be able to combine everything into a common class

File: tests/test_securitysystemthingspeak.py [hunk 4]
```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+"""
+ThingSpeak tests
+"""
+from __future__ import absolute_import
```
Comment (marianarafaelwhite) [inline tests/test_securitysystemthingspeak.py]: What's this used for?

File: tests/test_lightclapper.py [hunk 5]
```diff
@@ -1 +0,0 @@
-# lightclapper unit tests
```
Comment (marianarafaelwhite) [inline tests/test_lightclapper.py]: I think removing this and the tempsensor version might cause merge conflicts

**Review by marianarafaelwhite — APPROVED** [review]
Looks good to me! I don't know if anything is missing and all of my comments are minor.

The pan tilt stuff looks cool

File: securitysystem/pantilt.py (hunk 2 above)
Comment (Hasan-Baig) [reply securitysystem/pantilt.py]: im not sure yet but most likely removed

File: securitysystem/pantilt.py (hunk 1 above)
Comment (Hasan-Baig) [reply securitysystem/pantilt.py:151]: resolved

File: securitysystem/pantilt.py (hunk 2 above)
Comment (Hasan-Baig) [reply securitysystem/pantilt.py]: removed

File: securitysystem/thingspeakreader.py (hunk 3 above)
Comment (Hasan-Baig) [reply securitysystem/thingspeakreader.py:11]: I don't think we have much time left so might as well leave them separate, or else if we start combining them and there are bugs we will run into more problems

File: tests/test_securitysystemthingspeak.py (hunk 4 above)
Comment (Hasan-Baig) [reply tests/test_securitysystemthingspeak.py]: removed

File: securitysystem/thingspeakreader.py (hunk 3 above)
Comment (marianarafaelwhite) [reply securitysystem/thingspeakreader.py:11]: I agree, it can be something we talk about in our report instead maybe

**Pull request merged**